pages/linux/edquota.md

```diff
@@ -0,0 +1,28 @@
+# edquota
+
+> Edit quota for a user or group. By default it operates on all file systems with quotas.
+> Quota information is stored permanently in the quota.user and quota.group files in the root of the file system.
+
+- Edit quota of current user (default):
+
+`edquota --user {{user}}`
```

Comment:
Sorry, I guess I wasn't clear :( I actually meant removing `--user {{username}}` altogether, assuming that the tool would run for the current user if no user is specified. Would the simple command without any parameters work both in `edquota` and `repquota`?